minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java

```diff
@@ -53,6 +53,16 @@ public AbstractContainer(MesosCluster cluster, String uuid, String containerId,
     }
 
     /**
+     * Returns lenght of the give statement
+     *
+     * @param claim string to check the length
+     * @return
+     */
+    private int getLengthOrNpe(String statement) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make "getLengthOrNpe" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2325)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'getLengthOrNpe' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUnusedPrivateMethod)